buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1498,6 +1498,7 @@ artifactId=${project.name}
         testClassesDirs = project.files(project.project(":beam-sdks-java-core").sourceSets.test.output.classesDirs, project.project(":beam-runners-core-java").sourceSets.test.output.classesDirs)
         maxParallelForks config.parallelism
         useJUnit(config.testCategories)
+        dependsOn ':beam-sdks-java-container:docker'
```

Comment:
Is this later going to change to use the process environment?

Reply:
I will explore the benefit of moving it to process environment. There is no real benefit of moving it to process based environment as of now.